output/influxdb/config.go

```diff
@@ -161,20 +155,6 @@ func ParseURL(text string) (Config, error) {
 			default:
 				return c, fmt.Errorf("insecure must be true or false, not %s", vs[0])
 			}
-		case "payload_size":
```

Comment:
hm I guess you fixed #2137  through dropping it @codebien :rofl:

Reply:
It was used for the UDP client that has been dropped in the v2.

Reply:
I guess this is another breaking change, can you please make a list of the breaking changes in the main PR message ?